esmvaltool/recipes/recipe_climwip_brunner2019.yml

```diff
@@ -0,0 +1,351 @@
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+  - {dataset: IPSL-CM5A-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r4i1p1}
+
+  - {dataset: IPSL-CM5A-MR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: IPSL-CM5B-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MIROC-ESM-CHEM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MIROC-ESM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  # MIROC5 has extended historical runs (overlap with RCP85) the pre-processor can't merge
+  # the first ensemble member for rsus (for the other ones it works)
+  # - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: MIROC5, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: MPI-ESM-LR, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+
+  - {dataset: MPI-ESM-MR, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MRI-CGCM3, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: MRI-ESM1, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: NorESM1-ME, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: NorESM1-M, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+
+obs_data: &obs_data  # for climwip performance metrics
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3}
+
+preprocessors:
+  climwip_general: &general
+    regrid:
+      target_grid: 2.5x2.5
+      scheme: linear
+    mask_landsea:
+      mask_out: sea
+
+  climatological_mean: &mean
+    climate_statistics:
+      operator: mean
+
+  climatological_std: &std
+    climate_statistics:
+      operator: std_dev
+
+  region_eur: &eur
+    <<: *general
+    extract_region:
+      start_longitude: -10.0
+      end_longitude: 39.0
+      start_latitude: 30.0
+      end_latitude: 76.25
+
+  region_neu: &neu
+    <<: *general
+    extract_shape:
+      shapefile: 'shapefiles/SREX/NEU/NEU.shp'
+      method: contains
+      crop: true
+
+  region_ceu: &ceu
+    <<: *general
+    extract_shape:
+      shapefile: 'shapefiles/SREX/CEU/CEU.shp'
+      method: contains
+      crop: true
+
+  region_med: &med
+    <<: *general
+    extract_shape:
+      shapefile: 'shapefiles/SREX/MED/MED.shp'
+      method: contains
+      crop: true
+
+  # --- optional: comment in **one ** target variable here --------------------
+  # this is only needed if no performance sigma is given
+  # !! need to comment in CLIM_future below !!
+  target: &target
+    short_name: tas
+    # short_name: pr
```

Comment:
Why not move this to the block where it's used? It's not repeated anywhere else, is it?

Reply:
Indeed my motivation here was just to only have one place in the recipe where things have to be changed. This avoids scrolling around in the recipe and minimizes the risk of one step being overlooked. But I'm find with either way.

Reply:
Then I would put all those settings all the way on top, even before the datasets section.